src/components/ProductTour/ProductTour.scss

```diff
@@ -0,0 +1,51 @@
+.ProductTour {
+    height: 600px;
+    width: 800px;
+
+    &__slide {
+        padding: 25px;
+        height: 540px;
+        overflow: auto;
+        box-sizing: border-box;
+    }
+
+    &__footer {
+        position: absolute;
+        bottom: 0;
```

Comment:
Not sure that `position: absolute`, `bottom: 0` and `width: 100%` are necessary.
The previous sibling will already push the footer down since it has a fixed height.
And the footer seems to be a block level element, so it should take up full width without explicit definition.

Reply:
I had it without it, and this solution seemed to me a pix better, but indeed it would work without it as well